exe/RadicleExe.hs

```diff
@@ -90,4 +91,4 @@ opts = Opts
 createBindings :: (MonadIO m, ReplM m) => [Text] -> IO (Bindings (PrimFns m))
 createBindings scriptArgs' = do
     evalServerPackendPrimFns <- createEvalServerBackendPrimFns
-    pure $ addPrimFns (replPrimFns scriptArgs' <> evalServerPackendPrimFns <> ipfsPrimFns) pureEnv
+    pure $ addPrimFns (replPrimFns scriptArgs' <> evalServerPackendPrimFns <> ipfsPrimFns <> daemonClientPrimFns) pureEnv
```

Comment:
Shouldn't we just remove the server stuff?

Reply:
And shouldn't we also remove the `ipfsPrimFns`?

Reply:
(As well as all of `MachineBackend/Ipfs.hs`)

Reply:
Yes! But instead of doing it in this PR we should tackle this separately in a cleanup PR.

Reply:
:+1:. My PR is removing any use of these functions, so I can do it.